project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_forms.scss

```diff
@@ -93,6 +93,81 @@
   text-transform: uppercase;
 }
 
+// ========================================================================== //
+// Event Form
+// ========================================================================== //
+.node-form {
+  .control-label {
+    clear: both;
+    display: block;
+  }
+
+  .form-control:not(textarea) {
+    color: $color-gray-dark;
+    font-style: italic;
+    background-color: #EBEAE4;
+    border: 0;
+    border-radius: 20px;
+
+    &:focus {
+      box-shadow: 0 0 8px rgba(0, 0, 0, .6);
+    }
+  }
+
+  .document_file {
+    display: none !important;
+  }
+
+  .hidden-input {
```

Comment:
@guytepper 
I'm assuming that `.hidden-input` is attached to an input with `type="hidden"` which is not displayed anyway on the DOM, hence, what is the use case for this one ?